deploy/helm/kubecf/templates/db-seeder.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if .Values.features.embedded_database.enabled -}}
+
```

Comment:
Could you declare the databases here like in https://github.com/SUSE/kubecf/blob/bde658ee927d59f41158f94bb90dcd6161ce75b4/deploy/helm/kubecf/assets/operations/buildpacks/set_suse_buildpacks.yaml#L9-L18?

This way we can do a loop for declaring the volumes and the mounts from a single source.

Reply:
done